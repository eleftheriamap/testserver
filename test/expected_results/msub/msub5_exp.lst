
test/exp/test/gen/msub/msub5_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	111aad08 	add	w8, w8, #0x6ab
   4:	510c6718 	sub	w24, w24, #0x319
   8:	d14482b5 	sub	x21, x21, #0x120, lsl #12
   c:	9b18d503 	msub	x3, x8, x24, x21
  10:	8a000000 	and	x0, x0, x0
