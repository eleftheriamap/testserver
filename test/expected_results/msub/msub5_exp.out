Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = fffff9550002a7b3
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 00000000000006ab
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = ffffffffffee0000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 00000000fffffce7
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x111aad08
0x00000004 : 0x510c6718
0x00000008 : 0xd14482b5
0x0000000c : 0x9b18d503
0x00000010 : 0x8a000000
